src/symbolic_memory.ml

```diff
@@ -20,16 +22,28 @@ module M = struct
     { data : (Int32.t, Expr.t) Hashtbl.t
     ; parent : t Option.t
     ; mutable size : int
+    ; chunks : (Int32.t, Expr.t) Hashtbl.t
     }
 
   let create size =
-    { data = Hashtbl.create 128; parent = None; size = Int32.to_int size }
+    { data = Hashtbl.create 128
+    ; parent = None
+    ; size = Int32.to_int size
+    ; chunks = Hashtbl.create 0
```

Comment:
You can put `Hashtbl.create 16` here, it's what the stdlib is going to do anyway.